src/plugins/intel_gpu/src/kernel_selector/kernels/fully_connected/fully_connected_kernel_bf_tiled.cpp

```diff
@@ -444,10 +485,24 @@ JitConstants FullyConnected_bf_tiled::GetJitConstants(const fully_connected_para
         jit.AddConstant(MakeJitConstant("FILTER_LOAD_BLOCK_SIZE", block_read_size));
         jit.AddConstant(MakeJitConstant("FILTER_ELEMENTS_PER_LOAD", weights_elements_per_load));
         jit.Merge(make_int4_packed_type_jit_constant("INT4_PACKED_TYPE_PRELOAD", params.weights.GetDType(), weights_elements_per_load));
+    } else {
+        jit.AddConstant(MakeJitConstant("USE_SLM", 0));
+    }
+
+    // Validated perf gain, Dynamic quantize force enable SCALE_POST_OP for char type multiplication
+    if (is_dynamic_quantize(params) && dispatchData.tile_m > 1 && dispatchData.tile_n == 2) {
+        jit.AddConstant(MakeJitConstant("DYNAMIC_QUANTIZE", 1));
+        jit.AddConstant(MakeJitConstant("DECOMPRESSION_SCALE_POST_OP", 1));
+    } else {
+        jit.AddConstant(MakeJitConstant("DYNAMIC_QUANTIZE", 0));
     }
 
+    jit.AddConstant(MakeJitConstant("DQ_TYPE", "char"));
+    jit.AddConstant(MakeJitConstant("QUANTIZE_GROUP_SIZE", quantize_grp_size));
```

Comment:
I'd suggest to define these only when `DYNAMIC_QUANTIZE` is on.

Reply:
Moved them.